tests/components/matter/test_select.py

```diff
@@ -198,3 +198,22 @@ async def test_map_select_entities(
     await trigger_subscription_callback(hass, matter_client)
     state = hass.states.get("select.laundrywasher_number_of_rinses")
     assert state.state == "normal"
+
+
+@pytest.mark.parametrize("node_fixture", ["pump"])
+async def test_pump(
+    hass: HomeAssistant,
+    matter_client: MagicMock,
+    matter_node: MatterNode,
+) -> None:
+    """Test MatterMapSelectEntity entities are discovered and working from a laundrywasher fixture."""
```

Comment:
nit: docstring mentions entity class and fixture from previous test
```suggestion
    """Test MatterAttributeSelectEntity entities are discovered and working from a pump fixture."""
```

Reply:
Thanks. Fixed